{"uuid":"3b8e5739-05b9-4f47-8409-5aa2d1f8b550","historyId":null,"status":"broken","statusDetails":{"message":"Oops...we found an error preparing this test file:\n\n  > cypress\\e2e\\tests\\login.cy.js\n\n","trace":"\n\nError: Webpack Compilation Error\n./cypress/pages/LoginPage.js\nModule build failed (from C:/Users/Hp/AppData/Local/Cypress/Cache/10.4.0/Cypress/resources/app/node_modules/babel-loader/lib/index.js):\nSyntaxError: D:\\Faiza\\gitHubProjectsLearning\\CypressFramework\\cypress\\pages\\LoginPage.js: Identifier 'generics' has already been declared. (5:7)\n\n  3 | export const configData = require('../fixtures/config.json')\n  4 | export const loginData = require('../fixtures/login.json')\n> 5 | import generics from \"../helpers/generics\"\n    |        ^\n  6 | const helper = new generics();\n  7 | export default class LoginPage{\n  8 |\n @ ./cypress/e2e/tests/login.cy.js 5:40-72\n    at handle (C:\\Users\\Hp\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\@cypress\\webpack-preprocessor\\dist\\index.js:180:23)\n    at finalCallback (C:\\Users\\Hp\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:257:39)\n    at C:\\Users\\Hp\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:306:14\n    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\\Users\\Hp\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at AsyncSeriesHook.lazyCompileHook (C:\\Users\\Hp\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\Hook.js:154:20)\n    at C:\\Users\\Hp\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:304:22\n    at Compiler.emitRecords (C:\\Users\\Hp\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:499:39)\n    at C:\\Users\\Hp\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:298:10\n    at C:\\Users\\Hp\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:485:14\n    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\\Users\\Hp\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at AsyncSeriesHook.lazyCompileHook (C:\\Users\\Hp\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\Hook.js:154:20)\n    at C:\\Users\\Hp\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:482:27\n    at C:\\Users\\Hp\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\neo-async\\async.js:2818:7\n    at done (C:\\Users\\Hp\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\neo-async\\async.js:3522:9)\n    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\\Users\\Hp\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at AsyncSeriesHook.lazyCompileHook (C:\\Users\\Hp\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\Hook.js:154:20)\n    at C:\\Users\\Hp\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:464:33\n    at C:\\Users\\Hp\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\graceful-fs\\graceful-fs.js:143:16\n    at C:\\Users\\Hp\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\graceful-fs\\graceful-fs.js:143:16\n    at C:\\Users\\Hp\\AppData\\Local\\Cypress\\Cache\\10.4.0\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\graceful-fs\\graceful-fs.js:61:14\n    at FSReqCallback.oncomplete (fs.js:146:23)\n\nThis occurred while Cypress was compiling and bundling your test code. This is usually caused by:\n\n- A missing file or dependency\n- A syntax error in the file or one of its dependencies\n\nFix the error in your code and re-run your tests."},"stage":"finished","attachments":[{"name":"video recording","type":"video/mp4","source":"8033f884-0e4d-4165-8031-d636874deee3-attachment.mp4"}],"parameters":[],"labels":[{"name":"suite","value":"cypress/e2e/tests/login.cy.js"}],"links":[],"start":1660395850977,"name":"Oops...we found an error preparing this test file: cypress/e2e/tests/login.cy.js","fullName":"Oops...we found an error preparing this test file: cypress/e2e/tests/login.cy.js","stop":1660395850977}